docs/index.txt

```diff
@@ -13,14 +13,14 @@ Laravel MongoDB
    :titlesonly:
    :maxdepth: 1
 
-   /install
```

Comment:
Including this fix so I can close [this PR](https://github.com/mongodb/laravel-mongodb/pull/2743)